include/aos/common/types.hpp

```diff
@@ -36,6 +51,25 @@ constexpr auto cInstanceIDLen = AOS_CONFIG_TYPES_INSTANCE_ID_LEN;
  */
 constexpr auto cFilePathLen = AOS_CONFIG_TYPES_FILE_PATH_LEN;
 
+/*
+ * URL len.
+ */
+constexpr auto cURLLen = AOS_CONFIG_TYPES_URL_LEN;
+
+/**
+ * Vendor version len.
+ */
+constexpr auto cVendorVersionLen = AOS_CONFIG_TYPES_VENDOR_VERSION_LEN;
+
+/**
+ * Service/layer description len.
+ */
+
+constexpr auto cDescriptionLen = AOS_CONFIG_TYPES_DESCRIPTION_LEN;
+
+constexpr auto cSHA256Size = 64;
```

Comment:
Isn't the size of SHA256 32 bytes? Or is it intentionally specified differently?

Reply:
Mindlessly copied from grpc options.Fixed.